File: test/fixtures/ruby/get_with_header_without_value.rb
require 'net/http'

uri = URI('http://localhost:28139/get')
req = Net::HTTP::Get.new(uri)
req.content_type = 'text/xml;charset=UTF-8'
req['getWorkOrderCancel'] = ''

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/csharp/get_with_header_without_value.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/get");

request.Headers.Add("getWorkOrderCancel", "");

request.Content = new StringContent("");
request.Content.Headers.ContentType = MediaTypeHeaderValue.Parse("text/xml;charset=UTF-8");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();
